src/Utilities/tradingPairIcons.js

```diff
@@ -1,16 +1,83 @@
 import USD from "../../node_modules/cryptocurrency-icons/svg/color/usd.svg";
 import BTC from "../../node_modules/cryptocurrency-icons/svg/color/btc.svg";
+import ETH from "../../node_modules/cryptocurrency-icons/svg/color/eth.svg";
+import LTC from "../../node_modules/cryptocurrency-icons/svg/color/ltc.svg";
+import BCH from "../../node_modules/cryptocurrency-icons/svg/color/bch.svg";
+import EOS from "../../node_modules/cryptocurrency-icons/svg/color/eos.svg";
+import XRP from "../../node_modules/cryptocurrency-icons/svg/color/xrp.svg";
+import DASH from "../../node_modules/cryptocurrency-icons/svg/color/dash.svg";
+import BNB from "../../node_modules/cryptocurrency-icons/svg/color/bnb.svg";
+import XMR from "../../node_modules/cryptocurrency-icons/svg/color/xmr.svg";
+import VET from "../../node_modules/cryptocurrency-icons/svg/color/vet.svg";
+import TRX from "../../node_modules/cryptocurrency-icons/svg/color/trx.svg";
+import LINK from "../../node_modules/cryptocurrency-icons/svg/color/link.svg";
+import TOMO from "../../node_modules/cryptocurrency-icons/svg/color/tomo.svg";
 
```

Comment:
Good demonstration of bringing in the resource of SVG icons that you demonstrated earlier and execution of population by the switch statement you walked us through.